bundle/Controller/Controller.php

```diff
@@ -4,12 +4,15 @@
 
 namespace Netgen\Bundle\EzPlatformSiteApiBundle\Controller;
 
-use eZ\Bundle\EzPublishCoreBundle\Controller as BaseController;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\Templating\GlobalHelper;
 use eZ\Publish\Core\QueryType\QueryTypeRegistry;
 use Netgen\EzPlatformSiteApi\API\Site;
 use Netgen\EzPlatformSiteApi\API\Values\Location;
 use Netgen\EzPlatformSiteApi\Core\Traits\PagerfantaTrait;
 use Netgen\EzPlatformSiteApi\Core\Traits\SearchResultExtractorTrait;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller as BaseController;
```

Comment:
This should be `Symfony\Bundle\FrameworkBundle\Controller\AbstractController` as the one here is deprecated in 4.2.

https://symfony.com/blog/new-in-symfony-4-2-important-deprecations#deprecated-the-base-controller-class

`AbstractController` is available in Symfony 3.4, so no issues there.

Reply:
Hm... It does restrict the usage of `$this->get()`, but there is a way around it too, as far as I can see.

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/115/commits/af471d846009e3e255488d1b015e936d69f2e285

Reply:
Missed your 2nd comment, not reverted the commit from above as discussed in person. We'll update for 4.2 later.